doc/src/manual/arrays.md

```diff
@@ -413,9 +413,12 @@ julia> sum(1/n^2 for n=1:1000)
 When writing a generator expression with multiple dimensions inside an argument list, parentheses
 are needed to separate the generator from subsequent arguments:
 
-```julia-repl
+```jldoctest
 julia> map(tuple, 1/(i+j) for i=1:2, j=1:2, [1:4;])
-ERROR: syntax: invalid iteration specification
+ERROR: ParseError:
+# Error @ none:1:44
```

Comment:
is this line number `none:1:44` stable?

Reply:
I think it is stable because `1` is just "the current line" and `44` is the character offset where the error is, which also is fixed here as long as the input stays fixed.